<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>42</y>
      <width>30</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>73</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>104</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Bits</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialportCBox">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>42</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="baudrate_CBox">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>73</y>
      <width>77</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="databits_ComboBox">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>104</y>
      <width>38</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="stopbitsCBox">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>135</y>
      <width>49</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="serialparityCBox">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>166</y>
      <width>107</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>none</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odd Parity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Even Parity</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="clearbitsButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>311</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Receive</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rev_time_checkbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>342</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Timesatmp</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rev_hex_checkbox">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>312</y>
      <width>108</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hex Receive</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rev_line_checkbox">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>342</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Wrap</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="recvEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>481</width>
      <height>351</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="serialdetectbutton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>231</y>
      <width>129</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Serial Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serialopenbutton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Serial Port</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial Port Settings</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Receive Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>135</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Bits</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>166</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Send Settings</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>20</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="send_hex_checkbox">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>410</y>
      <width>89</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hex Send</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="send_line_checkbox">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>410</y>
      <width>126</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send New Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendbutton">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>421</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtSendMs">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>439</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="TimeSend_checkbox">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>440</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clrsendbutton">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>452</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Send</string>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>serialportCBox</zorder>
   <zorder>baudrate_CBox</zorder>
   <zorder>databits_ComboBox</zorder>
   <zorder>stopbitsCBox</zorder>
   <zorder>serialparityCBox</zorder>
   <zorder>clearbitsButton</zorder>
   <zorder>rev_time_checkbox</zorder>
   <zorder>rev_hex_checkbox</zorder>
   <zorder>rev_line_checkbox</zorder>
   <zorder>recvEdit</zorder>
   <zorder>sendEdit</zorder>
   <zorder>serialdetectbutton</zorder>
   <zorder>serialopenbutton</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>send_hex_checkbox</zorder>
   <zorder>send_line_checkbox</zorder>
   <zorder>sendbutton</zorder>
   <zorder>txtSendMs</zorder>
   <zorder>TimeSend_checkbox</zorder>
   <zorder>clrsendbutton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
